### ISSUE TITLE
Publish contacts plugin on website

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo.github.io#202]
As the plugin is currently not available on Play we could at least offer it to download on our website, for those specifically interested.

Side note:
Linking to this download from within main app not allowed as per Google Play rules.

**Comment by kumy** [comment]
Regarding fdroid, the link used to download this apk is https://github.com/cgeo/cgeo/releases/download/market_20150112/cgeo-contacts_v1.5.apk

https://github.com/cgeo/cgeo-fdroid-base/blob/f6e774efd2c66814a7042802d0d3fb553a998ef9/scripts/fdroid-update.sh#L98

Is this still the right file?

**Comment by Lineflyer** [comment]
Version code looks good, will doublecheckVon meinem Smartphone gesendet.
null

**Comment by bekuno** [comment]
It seems that this change was not done.
The current link to the contacts plugin at https://www.cgeo.org/development refers to
http://download.cgeo.org/cgeo-contacts-nightly.apk
where nothing is to find.

**Comment by Lineflyer** [comment]
@kumy How to update the fdroid version of the contacts plugin?. Is it sufficient to update the file you mentioned above and run the fdroid-release job on CI?

**Comment by kumy** [comment]
Yes update the script is the first step, then I need to update the docker stack on my side, then we can trigger updtaes via Jenkins. @Lineflyer, would you please do the file file updtae? I ll deploy on sunday evening. Thanks

**Comment by bekuno** [comment]
I edited the PR https://github.com/cgeo/cgeo-fdroid-base/pull/2 for the new plugin version.

**Comment by bekuno** [comment]
The download link at the website https://www.cgeo.org/development.html#nightly-builds does not work:
http://download.cgeo.org/cgeo-contacts-nightly.apk
We should fix or remove it.

**Comment by kumy** [comment]
Are we still providing this plugin?
